examples/webgl_lights_spotlight.html

```diff
@@ -52,6 +59,23 @@
 				var ambient = new THREE.AmbientLight( 0xffffff, 0.1 );
 				scene.add( ambient );
 
+				var textureLoader = new THREE.TextureLoader();
+				textureUrls = [ 'none', 'sintel (video)', 'uv_grid_opengl.jpg', 'sprite2.png', 'colors.png' ];
+				textures = { none: null };
+
+				var video = document.getElementById( 'video' );
+				textures[textureUrls[1]] = new THREE.VideoTexture( video );
+				textures[textureUrls[1]].minFilter = THREE.LinearFilter;
+				textures[textureUrls[1]].magFilter = THREE.LinearFilter;
```

Comment:
`VideoTextures`'s `minFilter` and `magFilter` have `THREE. LinearFilter` as default value.

Reply:
Filters removed

Reply:
removing the video texture from examples is also an editorial call, I'll remove it if needed